frontend/src/v5/ui/components/viewer/cards/cardFilters/filterForm/filterFormValues/filterFormValues.component.tsx

```diff
@@ -106,6 +122,18 @@ export const FilterFormValues = ({ type }: { type: CardFilterType }) => {
 			</>
 		);
 	}
+	if (isSelectType(type)) {
+		return (
+			<FormMultiSelect
+				name={name}
+				formError={error?.[0]}
+				transformValueIn={mapFormArrayToArray}
+				transformChangeEvent={(e) => mapArrayToFormArray(compact(e.target.value))}
```

Comment:
Can you rename these to "transformInput" and "transformOuput"?

Reply:
That's what I initially did until I realised it might get confusing as in InputController there is another prop called `Input` and `transformInput` would not transform this `Input`
Perhaps `transformInputValue`/`transformOutputValue` would solve this confusion